server/main.go

```diff
@@ -18,8 +16,8 @@ type ServerConfig struct {
 	Tables []*schema.Table
 }
 
-//go:embed templates/*
-var content embed.FS
+//go:embed web/dist
```

Comment:
The linter fails on this line because web/dist is empty.
It's empty because the dist files are not pushed to git.
Should those files be pushed? Could that lint rule be omitted?